docs/api-examples/resend.mdx

```diff
@@ -29,7 +29,7 @@ superface login
 Resend have an [OAS version](https://raw.githubusercontent.com/resendlabs/resend-openapi/main/resend.yaml) of their API, but their [API Reference](https://resend.com/docs/api-reference/introduction) can also be used without any additional changes so you can copy and paste the URL for the `send email` endpoint.
```

Comment:
I feel like this sentence isn't valid any more. The reference CANNOT be actuallyy used :(

Reply:
Good catch